src/patch_parse.c

```diff
@@ -74,7 +74,10 @@ static int parse_advance_expected(
 	return 0;
 }
 
-#define parse_advance_expected_s(ctx, str) \
+#define parse_advance_expected_str(ctx, str) \
+	parse_advance_expected(ctx, str, strlen(str))
+
+#define parse_advance_expected_const_str(ctx, str) \
```

Comment:
Is this distinction really required? To the best of my knowledge, most compilers convert `strlen("CONST_STRING")` to a compile-time constant anyway.